### PR TITLE
feat: upgrade dependencies

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: packages/frontend/tailwind.config.cjs [hunk 1]
```diff
@@ -12,5 +10,4 @@ module.exports = {
     corePlugins: {
         preflight: false,
     },
-    plugins: [require("@tailwindcss/line-clamp")],
```
Comment (guerrap) [inline packages/frontend/tailwind.config.cjs:15]: Are we sure we don't need this plugin anymore?

File: packages/frontend/src/creation-form/components/oracles-configuration/oracle-creation-form/index.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import { OracleCreationFormWrapper } from "../oracle-creation-form-wrapper";
+import {
+    NamespacedTranslateFunction,
+    OracleCreationFormProps,
+    OracleInitializationBundleGetter,
+    useResolvedTemplate,
+} from "@carrot-kpi/react";
+import { AugmentedOracleDataMap } from "..";
+
+interface OraclesAccordionProps {
```
Comment (guerrap) [inline packages/frontend/src/creation-form/components/oracles-configuration/oracle-creation-form/index.tsx:12]: This should be `OracleCreationFormProps` I guess.

File: packages/frontend/tailwind.config.cjs (hunk 1 above)
Comment (luzzif) [reply packages/frontend/tailwind.config.cjs:15]: It was included in the core of the upgraded version of Tailwind. I got a warning right after upgrading so I removed it.

File: packages/frontend/src/creation-form/components/oracles-configuration/oracle-creation-form/index.tsx (hunk 2 above)
Comment (luzzif) [reply packages/frontend/src/creation-form/components/oracles-configuration/oracle-creation-form/index.tsx]: Addressed in bd61c1a740b3c01f9150fa350084963893855b0e

**Review by guerrap — APPROVED** [review]

**Pull request merged**